code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -1,19 +1,19 @@
 /datum/outfit/plasmaman/bar
 	name = "Bartender Plasmaman"
 
-	head = /obj/item/clothing/head/helmet/space/plasmaman/white
+	head = /obj/item/clothing/head/helmet/space/plasmaman
```

Comment:
All of the `/datum/outfit/plasmaman`'s can be removed, since it's inherited from `/datum/outfit/plasmaman`
https://github.com/yogstation13/Yogstation-TG/blob/bd36ec2b9a5b9b2fddc17649ca697960f4f999f3/code/modules/clothing/outfits/standard.dm#L366